package.json

```diff
@@ -60,7 +59,6 @@
     "README.md",
     "index.js",
     "cjs/",
-    "esm/",
```

Comment:
I think it's fine to keep the folder but let's just remove the `module` thing until the story is more fleshed out.